### PR TITLE
feat: simplify login flow

### PR DESCRIPTION
**Pull request by cmoog**
## When browser is openable

Screen Shot 2021-03-09 at 12 02 38 PM

## When open in browser fails

Screen Shot 2021-03-09 at 12 06 55 PM

## Token copy page
Screen Shot 2021-03-09 at 10 35 42 AM

### CODE REVIEW

File: internal/cmd/login.go [hunk 1]
```diff
@@ -42,25 +39,51 @@ func loginCmd() *cobra.Command {
...
+	}
+
+	token := readLine("Paste token here: ")
+	if err := pingAPI(ctx, envURL, token); err != nil {
+		return xerrors.Errorf("ping API with credentials: %w", err)
+	}
+	return storeConfig(envURL, token, config.URL, config.Session)
+}
+
+func readLine(prompt string) string {
```
Comment (vapurrmaid) [inline internal/cmd/login.go:82]: I feel like we can break this out from `login.go`, but that can be done later

**Review by vapurrmaid — APPROVED** [review]
Awesome stuff :100:

**Review by fuskovic — APPROVED** [review]
I like it. 

Reminds me of the gcloud auth flow.

**Comment by cmoog** [comment]
Oh we'll need to note this in the change log for 1.17 since v1.17 of the CLI will required v1.17 of the API. Standard stuff

**Comment by vapurrmaid** [comment]
> Oh we'll need to note this in the change log for 1.17 since v1.17 of the CLI will required v1.17 of the dashboard. Standard stuff

Might be worth adding a ticket or task to https://app.clubhouse.io/coder/epic/6121/okr-3-1-establish-go-to-market-approach-for-new-product-and-feature-launches?vc_group_by=day somewhere

**Pull request merged**